### PR TITLE
Updated localization strings for Image Viewer

### PR DESCRIPTION
**Pull request by flukeout**

### CODE REVIEW

File: locales/en-US/editor.properties [hunk 1]
```diff
@@ -145,7 +145,9 @@ DND_SUCCESS_UNZIP=Successfully unzipped <b>{0}</b>.
 DND_SUCCESS_UNTAR=Successfully untarred <b>{0}</b>.
 
 # Image Viewer
-IMAGE_DIMENSIONS={0} (width) &times; {1} (height) pixels
+IMAGE_FILE_TITLE=Image File
+IMAGE_DIMENSIONS={0} &times; {1} pixels
```
Comment (humphd) [inline locales/en-US/editor.properties]: This is a change, so I think we need a new string key.  cc @TheoChevalier

**Comment by flukeout** [comment]
Okay, will await Theo's instructions.

File: locales/en-US/editor.properties (hunk 1 above)
Comment (TheoChevalier) [inline locales/en-US/editor.properties]: Yes please, it needs to be retranslated

**Comment by flukeout** [comment]
@humphd updated the string key name

File: locales/en-US/editor.properties [hunk 2]
```diff
@@ -145,7 +145,9 @@ DND_SUCCESS_UNZIP=Successfully unzipped <b>{0}</b>.
 DND_SUCCESS_UNTAR=Successfully untarred <b>{0}</b>.
 
 # Image Viewer
-IMAGE_DIMENSIONS={0} (width) &times; {1} (height) pixels
+IMAGE_FILE_TITLE=Image File
+IMAGE_DIMENSIONS_1={0} &times; {1} pixels
```
Comment (humphd) [inline locales/en-US/editor.properties]: I would use `IMAGE_PIXEL_INFO` vs. a versioned one, but it's up to you.  Make sure whatever you use gets changed in Brackets too.

**Comment by humphd** [comment]
@gideonthomas make sure you're cool with this approach (e.g., removing the old, versioning the new) and then this is good to go from my end.

File: locales/en-US/editor.properties (hunk 2 above)
Comment (gideonthomas) [inline locales/en-US/editor.properties]: I think the l10n guidelines support your suggestion - create a meaningful name vs. just tacking on numbers.

**Comment by gideonthomas** [comment]
Yeah I think this looks good

**Comment by flukeout** [comment]
Okay cool, I'm adding an "Audio Viewer" so I'll update this with those strings as well.

**Pull request merged**